addon/templates/components/frost-expand.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="frost-expand-label"  data-test={{hook (concat hook '-label')}} onClick={{action '_onClick'}}>
+  {{frost-icon
+    class='frost-expand-label-icon'
+    hook=(concat hook '-label-icon')
+    pack='frost'
+    icon='expand-collapse'
+  }}
+  <div class="frost-expand-label-text" data-test={{hook (concat hook '-label-text')}}>
+    {{#if expanded}}
+      {{if collapseLabel collapseLabel 'Collapse'}}
```

Comment:
I made a comment above, but the default of `'Collapse'` makes the most sense to put in the default values in the prop-types area of the code so that all the information is centralized